.github/workflows/ci.yml

```diff
@@ -0,0 +1,130 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Should we be using `pull_request_target` instead of `pull_request` given the there are secrets in this file?

Reply:
@kevinbarabash AFAIK, secrets aren't and shouldn't be passed to forked repository.

Reply:
It's now possible to run Browserstack tests, e.g., Safari screenshotter, upon maintainer's approval via adding `test screenshots` label.

Reply:
That's super cool.